ui/app/src/modules/Search/SearchApp.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import { HashRouter, Route } from 'react-router-dom';
+import Search from './Search';
+
+
+function SearchApp(props: any) {
+  const {onEdit, onDelete, onPreview, onSelect, siteId} = props;
```

Comment:
If you just want to passthrough all SearchApp props down, simply rename one of the props params and spread them both.

Reply:
e.g.
```js
function SearchApp(topProps: any) {
// ...
<Route render={(renderProps) => <Search {..renderProps} {...topProps} />} />
// ...
```